plugins/ocm-backend/config.d.ts

```diff
@@ -0,0 +1,40 @@
+interface KubernetesPluginRef {
+  /**
+   * Match cluster name where the Open Cluster Management operator is installed from the kubernetes plugin configuration
+   */
+  kubernetesPluginRef: string;
+}
+
+interface HubClusterConfig {
+  /**
+   * Url of the hub cluster API endpoint
+   */
+  url: string;
+  /**
+   * Name of the cluster
+   */
+  name: string;
+  /**
+   * Service Account Token which is used for querying data from the hub
+   * @visibility secret
+   */
+  serviceAccountToken?: string;
```

Comment:
Doesn't this have to be required?

Reply:
It's not required. If not defined the default context from `kubeconfig` is used. This is consistent with kubernetes plugin for cluster with `authProvider: 'serviceAccount'` and without `serviceAccountToken` defined.

Reply:
That is true totally forgot about that.